controllers/resize_pvc.go

```diff
@@ -152,7 +203,24 @@ func (r *MySQLClusterReconciler) needResizePVC(cluster *mocov1beta2.MySQLCluster
 	return pvcSet, true
 }
 
-func isUpdatingStatefulSet(sts *appsv1.StatefulSet) bool {
+func (r *MySQLClusterReconciler) isVolumeExpansionSupported(ctx context.Context, pvc *corev1.PersistentVolumeClaim) (bool, error) {
+	if pvc.Spec.StorageClassName == nil {
+		return false, nil
```

Comment:
If storageClassName is omitted, the default storage class should be checked.
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1

Reply:
If the default storage class exists, isn't it that `storageClassName` is set by admission? 
Is there a case where `storageClassName` is nil when the default storage class exists?

Reply:
Ah, that's correct.